FrontEnd.cpp

```diff
@@ -52,15 +52,17 @@ using namespace Eigen;
 StaticFusion::StaticFusion(unsigned int res_factor)
 {
     //Resolutions and levels
-    rows = 480/res_factor;
-    cols = 640/res_factor;
-    fovh = M_PI*62.5/180.0;
-    fovv = M_PI*48.5/180.0;
-    width = 640/res_factor;
-    height = 480/res_factor;
+    width = Resolution::getInstance().width();
+    height = Resolution::getInstance().height();
+    cols = width;
+    rows = height;
+    cx = Intrinsics::getInstance().cx();
+    cy = Intrinsics::getInstance().cy();
```

Comment:
The variables `cx` and `cy` are nowhere used. Why are we setting this here? Shouldn't we set it where the centre of the image is used?